### PR TITLE
The root of all evil

### PR DESCRIPTION
**Pull request by darosior**
This implements daemonization, let's see if it builds under Windows. I'll test under a windows machine i could get my hand on tomorrow.

### CODE REVIEW

**Comment by darosior** [comment]
Works on Windows :tada:

**Comment by darosior** [comment]
Just needs a cap

File: src/main.rs [hunk 1]
```diff
@@ -1,4 +1,63 @@
...
+    Some(PathBuf::from(args[2].to_owned()))
+}
+
+fn daemon_main(revaultd: RevaultD) {}
+
+fn main() {
+    let args = env::args().collect();
+    let conf_file = parse_args(args);
+
+    match parse_config(conf_file) {
```
Comment (edouardparis) [inline src/main.rs:32]: for less indentation, I think you can do:
```suggestion
    let config = parse_config(conf_file).unwrap_or_else(|e| {
           eprintln!("Error parsing config: {}", e);
            process::exit(1);
    });
```

Comment (darosior) [reply src/main.rs:32]: Nice, thanks

**Review by edouardparis — APPROVED** [review]
ACK 8f8a06e025f9d66c93fb32d2fd33c8be7277c343

**Pull request merged**